frontend/src/App.tsx

```diff
@@ -26,8 +26,8 @@ import { UrlInputSection } from "./components/UrlInputSection";
 import TermsOfServiceDialog from "./components/TermsOfServiceDialog";
 
 function App() {
-  const [appState, setAppState] = useState<"INITIAL" | "CODING" | "CODE_READY">(
-    "INITIAL"
+  const [appState, setAppState] = useState<AppStatus>(
```

Comment:
In the future, would be good to keep names like this consistent: AppStatus -> AppState. I'll update